A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -681,6 +681,95 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
             this.updateAutopilotCooldown = this._apCooldown;
         }
     }
+    checkUpdateFlightPhase() {
+        const airSpeed = SimVar.GetSimVarValue("AIRSPEED TRUE", "knots");
+        if (airSpeed > 10) {
+            if (this.currentFlightPhase === 0) {
+                this.currentFlightPhase = FlightPhase.FLIGHT_PHASE_TAKEOFF;
+            }
+
+            //Changes to climb phase when acceleration altitude is reached
+            if (this.currentFlightPhase === FlightPhase.FLIGHT_PHASE_TAKEOFF) {
+                const agl = Simplane.getAltitudeAboveGround();
+                if (agl > this.accelerationAltitude) {
+                    this.currentFlightPhase = FlightPhase.FLIGHT_PHASE_CLIMB;
+                }
+            }
```

Comment:
there doesn't seem to be a default acceleration altitude, so many people on the dev branch (like me 😆 ) are getting stuck in takeoff mode and not knowing how to get out of it. Should we try to find a good way to det a feault accel altitude? or should something else be done? I think this will continue to trip up a lot of players, especially when spawning on runway. I think users should be able to spawn on the runway, turn off parking brake, and takeoff without any other changes IMO.

Reply:
this conversation might be best left to #369.